Content.Client/Guidebook/Controls/GuidebookControl.xaml

```diff
@@ -1,9 +1,14 @@
 ﻿<Control xmlns="https://spacestation14.io"
          xmlns:fancyTree="clr-namespace:Content.Client.UserInterface.Controls.FancyTree"
          xmlns:customControls="clr-namespace:Content.Client.Administration.UI.CustomControls"
+         xmlns:graphics="clr-namespace:Robust.Client.Graphics;assembly=Robust.Client"
          SetSize="750 700"
          MinSize="100 200">
-    <SplitContainer Orientation="Horizontal" HorizontalExpand="True" Name="Split">
+    <PanelContainer>
+        <PanelContainer.PanelOverride>
+            <graphics:StyleBoxFlat BackgroundColor="#25252A"/>
```

Comment:
Do not hardcode the bg color please

Reply:
A bunch of other UI's do this too. What should I do then

Reply:
I cant find another way to do this other then hardcoding it

Reply:
Fixed in [`a261437` (#20268)](https://github.com/space-wizards/space-station-14/pull/20268/commits/a261437c4bbf9776c0aeca5ce0c7a7c3388cfcf7)